server/controllers/finance/reports/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @overview finance/reports
+ *
+ * @description
+ * This module simply exposes the build() methods of the receipts and reports
+ * in the finance module.
+ */
+
+// expose to the express router
+module.exports = {
+  receipts : {
+    invoices: require('./invoice.receipt').build,
+
+    /** @todo - make this voucher receipt be exposed in this method */
```

Comment:
Yes.  In fact, I just added it.  @mbayopanda is currently working on the voucher report, and I didn't want to collide with him needlessly.

Eventually, this should read:
```js
receipts : {
   invoices: require('./invoice.receipt'),
   vouchers : require('./vouchers.receipt'),
}
```